source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -145,10 +156,64 @@ corresponding code.
          :end-before: end-change-stream-pipeline
          :language: csharp
 
-To learn more about modifying your change stream output, see the
-:manual:`Modify Change Stream Output
-</changeStreams/#modify-change-stream-output>` section in the {+mdb-server+}
-manual.
+Split Large Change Events Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your application generates change events that exceed 16 MB in size, the
+server returns a ``BSONObjectTooLarge`` error. To avoid this error, you can use
+the ``$changeStreamSplitLargeEvent`` pipeline stage to split the events
```

Comment:
Here we talk about the `$changeStreamSplitLargeEvent`, but we do not explicitly say that we have it as a step in the typed aggregation api, called `ChangeStreamSplitLargeEvent`. We mention this only in the code at the bottom of this section. I think it would be better to make this clear from the start